internal/pkg/manifest/env_test.go

```diff
@@ -522,6 +524,13 @@ http:
 			} else {
 				require.NoError(t, gotErr)
 				require.Equal(t, tc.wantedStruct, got)
+
+				roundtrip, err := yaml.Marshal(tc.wantedStruct)
+				require.NoError(t, err)
+				fmt.Printf("re-marshalled form:\n%s\n", string(roundtrip))
```

Comment:
could we just use `t.Logf()` instead so that the output isn't printed if the test passes?

Reply:
Good idea! I'll add that.

Reply:
Added!